<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetDetailedPageInfoDlg</class>
 <widget class="QDialog" name="GetDetailedPageInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get Detailed Page Info</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Menu Name(Menu ID)</string>
   </property>
   <widget class="QListWidget" name="listWidgetMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Details of Selected Item</string>
   </property>
   <widget class="QListWidget" name="listWidgetDetails">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetDesc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 600 16pt &quot;Segoe UI&quot;;

</string>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
